service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -48,10 +57,47 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             run2706(caseDetails);
         }
 
+        if ("FPLA-2722".equals(migrationId)) {
+            log.info("Performing migration ({}) for case {}", migrationId, caseDetails.getId());
+            run2417(caseDetails);
+        }
+
         caseDetails.getData().remove(MIGRATION_ID_KEY);
         return respond(caseDetails);
     }
 
+    private void run2417(CaseDetails caseDetails) {
+        CaseData caseData = getCaseData(caseDetails);
+
+        caseDetails.getData().put(
+            HEARING_FURTHER_EVIDENCE_DOCUMENTS_COLLECTION_KEY, caseData.getHearingFurtherEvidenceDocuments()
+        );
+        caseDetails.getData().put(
+            C2_DOCUMENTS_COLLECTION_KEY, caseData.getC2DocumentBundle()
+        );
+        caseDetails.getData().putAll(updateSupportingDocs(
+            caseData.getFurtherEvidenceDocuments(), FURTHER_EVIDENCE_DOCUMENTS_COLLECTION_KEY
+        ));
+        caseDetails.getData().putAll(updateSupportingDocs(
+            caseData.getFurtherEvidenceDocuments(), FURTHER_EVIDENCE_DOCUMENTS_COLLECTION_LA_KEY
+        ));
+        caseDetails.getData().putAll(updateSupportingDocs(
```

Comment:
is this correspondence of further evidence?

Reply:
good spot, bad copy and paste. Have updated tests to reveal this issue then fixed